Registry/registry.chem

```diff
@@ -20,6 +20,12 @@ state    real    QC_B4MP         ikj    misc        1         -      -        "Q
 state    real    QI_B4MP         ikj    misc        1         -      -        "QI_B4MP"             "CLOUD ICE BEFORE MICROPHYSICS"    "kg kg-1"
 state    real    QS_B4MP         ikj    misc        1         -      -        "QS_B4MP"             "SNOW BEFORE MICROPHYSICS"         "kg kg-1"
 
+state    real    ph_cw           ikj    misc        1         -      -        "ph_cw"               "pH of cloud water"             "unitless"
```

Comment:
Stacy,
If these diagnostics are only to be activated when a specific namelist option is selected, these should all be in a package.

Reply:
@ravanah @kayeekayee 
If this is indeed an optional diagnostic, then the namelist option that activates the pH for Mozart should be used for a package for the ph_cw, ph_aer01 through 04.